kornia/feature/dedode/dedode.py

```diff
@@ -0,0 +1,194 @@
+from typing import Dict, Optional, Tuple
+
+import torch
+import torch.nn.functional as F
+
+from kornia.augmentation._2d.intensity.normalize import Normalize
```

Comment:
The reason I used this, to avoid creating the mean-std tensor on device every time. But yes, circulars are much more important, I will use normalize from enhance for now

Reply:
I advocate for that utility class I mentioned or for next pr

Reply:
@edgarriba as I said in another comment - the utility class is already there %)